src/LibraryManager.Vsix/ErrorList/ErrorList.cs

```diff
@@ -35,28 +35,13 @@ public bool HandleErrors(IEnumerable<ILibraryOperationResult> results)
                     AddLineAndColumn(json, result.InstallationState, displayErrors);
 
                     Errors.AddRange(displayErrors);
-
-                    foreach (IError error in result.Errors)
-                    {
-                        Logger.LogEvent(error.Message, LogLevel.Operation);
```

Comment:
Moved the Logger call to Logger.LogEventsSummary
Moved Telemetry call to Telemetry.LogEventsSummary 

Both methods will log individual failures.